Examples/Copy-ISHCD.ps1

```diff
@@ -56,4 +58,27 @@ if($awss3)
     $key="$($awss3.ISHCDFolderKey)$($awss3.ISHCDFileName)"
     & $scriptsPaths\ISHServer\Copy-ISHCD.ps1 -Computer $computerName -Credential $credential -ISHServerVersion $ishServerVersion -BucketName $awss3.BucketName -Key $key -AccessKey $awss3.AccessKey -SecretKey $awss3.SecretKey
 }
-
+if($azurefilestorage)
```

Comment:
Looking at the rest of the code, I don't see the same conditional pattern. 
I would prefer the flow without the condition because:

- The **source** scripts should drive the intelligent part of how to use the **ISHServer** cmdlets
- The **example** scripts should be merely a map between a json file and the **source** scripts to execute.